modules/testbase.lua

```diff
@@ -128,6 +128,29 @@ function control.runNextCase()
     xmlReporter.AddCase(Test.current_case_name)
     atf_logger.LOGTestCaseStart(Test.current_case_name)
     testcase(Test)
+
+    local function wait(pConnection)
+      local timeout = config.unexpectedEventTimeout
+      local event = events.Event()
+      event.matches = function(e1, e2) return e1 == e2 end
+
+      local ret = Expectation("Wait", pConnection)
+      ret.event = event
+      event_dispatcher:AddEvent(pConnection, event, ret)
+      Test:AddExpectation(ret)
+      ret:Timeout(timeout + 5000)
```

Comment:
@dboltovskyi I would propose move all code which initialize `ret` before other actions with this expectation
```
      local ret = Expectation("Wait", pConnection)
      ret.event = event
      ret:Timeout(timeout + 5000)
      event_dispatcher:AddEvent(pConnection, event, ret)
      Test:AddExpectation(ret)
```

Reply:
@aderiabin Done in 0913260